exercises/bracket-push/README.md

```diff
@@ -7,7 +7,7 @@ verify that all the pairs are matched and nested correctly.
 
 Reading about these Rust topics may help you implement a solution.
 
-- Lifetimes and Structs: https://doc.rust-lang.org/book/lifetimes.html#impl-blocks
+- Lifetimes and Structs: https://doc.rust-lang.org/book/second-edition/ch10-03-lifetime-syntax.html#lifetime-annotations-in-struct-definitions
```

Comment:
So, it looks to me like the link to https://doc.rust-lang.org/book/lifetimes.html#impl-blocks (which would now be reachable at https://doc.rust-lang.org/book/first-edition/lifetimes.html#impl-blocks) is meant to help with `Brackets::from("[]")`. In light of that, might I suggest that https://doc.rust-lang.org/book/second-edition/ch10-03-lifetime-syntax.html#lifetime-annotations-in-method-definitions looks more relevant?

Reply:
I agree.